frontend/src/job/ingest-archival-material/IngestArchivalMaterialMetadataForm.vue

```diff
@@ -148,17 +148,22 @@ export default class ArchivalMaterialMetadataForm extends Vue {
 
 function evaluateTargetFolder(targetFolder : WorkbenchFileTree) {
     const errors: string[] = [];
-    if (!containsNumberOfFiles(targetFolder, 1)) {
+    if (containsNumberOfFiles(targetFolder, 0)) {
         errors.push(
-            `Folder has more than one entry. Only one subfolder 'tif' is allowed.`
+            `Folder appears to be empty. Please provide input data.`
         );
     }
 
-    if (!('tif' in targetFolder)) {
-        errors.push(`Folder does not have a subfolder 'tif'.`);
-    } else if (targetFolder.tif.contents !== undefined &&
-                !containsOnlyFilesWithSuffix(targetFolder.tif.contents, '.tif')) {
-        errors.push(`Subfolder 'tif' does not only contain files ending in '.tif'.`);
+    if (targetFolder !== undefined) {
+        // since there is no subfolder anymore get all files in question
+        let file_list = getVisibleFolderContents(targetFolder);
+        // check if everyone is a tif
+        let clear = file_list
+                    .map(file => file.name.endsWith('.tif') || file.name.endsWith('.tiff') )
+                    .reduce((a,b) => a && b, true);
+        if (!clear){
+            errors.push(`Folder does not only contain files ending in '.tif'.`);
+        }
```

Comment:
Ich würde den Aufruf von getVisibleFolderContents() tiefer schieben. Was spricht dagegen diesen Schnipsel mehr oder weniger 1:1 in die containsOnlyFilesWithSuffix() zu verschieben? Dann müsste man sich nicht auf individueller Job Ebene mit dem systeminternen, versteckten Dateien herumschlagen.